mail/rspamd/src/opnsense/mvc/app/models/OPNsense/Rspamd/RSpamd.xml

```diff
@@ -119,6 +119,13 @@
       <rewritesubject type="TextField">
         <Required>N</Required>
       </rewritesubject>
+      <historyrows type="IntegerField">
+        <default>200</default>
+        <Required>Y</Required>
+        <MinimumValue>1</MinimumValue>
+        <MaximumValue>100000</MaximumValue>
```

Comment:
This must be smaller since that can crash the browser. I suggest not to allow more than 3k lines

Reply:
Or is that backend only?

Reply:
Backend only, for Redis. The UI defaults to 25 rows per page. If this would crash the browser it would more likely be an upstream bug

Reply:
nope, I just know that form personal experience (fixing a bug in the nginx plugin) that if you have a row with 3k+ rows the browser will become extremely slow or crash totally due to the massive amount of data.